app/models/wish.rb

```diff
@@ -8,7 +8,7 @@ class Wish < ApplicationRecord
   has_rich_text :description
   has_many :comments, dependent: :destroy
 
-  enum stage: { Backlog: 0, 'In process': 1, 'In review': 2, Beta: 3, Launched: 4 }
```

Comment:
Do we need to make migration to update the existing `Beta` and `In Review` wishes?